ui/tests/acceptance/variables-test.js

```diff
@@ -983,4 +983,254 @@ module('Acceptance | variables', function (hooks) {
       });
     });
   });
+
+  module('Job Variables Page', function () {
```

Comment:
minor: I would expect these tests to be in a new `acceptance/job-variables-test.js` file to match the route hierarchy.